Avoid implicit function declarations and implicit ints for improved
compatibility with future compilers.

Submitted upstream: <https://github.com/xiaoyeli/superlu/pull/76>

diff --git a/EXAMPLE/cfgmr.c b/EXAMPLE/cfgmr.c
index 0f25f58923aa0c7e..d8f523f9b67fa5ca 100644
--- a/EXAMPLE/cfgmr.c
+++ b/EXAMPLE/cfgmr.c
@@ -33,6 +33,8 @@ For information on ITSOL contact saad@cs.umn.edu
 
 #define  epsmac  1.0e-16
 
+extern int ccopy_(int *, complex *, int *, complex *, int *);
+extern void caxpy_(int *, complex *, complex *, int *, complex *, int *);
 extern void cdotc_(complex *, int *, complex [], int *, complex [], int *);
 extern float scnrm2_(int *, complex [], int *);
 
